### DISCUSSION TITLE
jj commit -i vs. jj split -i

### DISCUSSION THREAD

**Discussion by WolfgangSteiner** [jj-vcs/jj#3589] [Q&A]
I am trying to understand if there are any meaningful differences between `jj commit -i` and `jj split -i` in order to break the current change into several commits. 

The help of `jj commit -i` says:
```  -i, --interactive        Interactively choose which changes to include in the first commit```

vs. `jj split -i`:
```  -i, --interactive          Interactively choose which parts to split. This is the default if no paths are provided```

In my current understanding, these two commands are identical for the task. Are there any situations when to prefer one over the other?

**Comment by martinvonz** [comment]
I think they are practically equivalent for splitting the working-copy commit (`jj split` can be used on any commit but `jj commit` is specifically for the working-copy commit). `jj commit -i` exists partly to help feel git and hg users feel more at home.
